smithy-protocols-traits/src/test/resources/software/amazon/smithy/protocols/traits/errorfiles/cbor-wire-format.errors

```diff
@@ -0,0 +1,5 @@
+[ERROR] smithy.example#InvalidService1: The `format` property for the smithy.protocols#rpcv2 protocol must have at least 1 element | Rpcv2Trait
+[ERROR] smithy.example#InvalidService2: The following values of the `format` property for the smithy.protocols#rpcv2 protocol are not supported: [invalid-wire-format] | Rpcv2Trait
+[ERROR] smithy.example#InvalidService3: The following values of the `format` property for the smithy.protocols#rpcv2 protocol are not supported: [invalid-wire-format] | Rpcv2Trait
+[ERROR] smithy.example#InvalidService4: The following values of the `format` property for the smithy.protocols#rpcv2 protocol are not supported: [invalid-wire-format1, invalid-wire-format2] | Rpcv2Trait
+[ERROR] smithy.example#InvalidService1: Error validating trait `smithy.protocols#rpcv2`: Missing required structure member `format` for `smithy.protocols#rpcv2` | TraitValue
```

Comment:
This doesn't seem like it should be happening. I think the `createNode` method on the trait class needs to serialize this field regardless of if it's empty. This should be fine; it'll only ever be valid with an entry.

Reply:
I am a little confused here: is there anything I should change? Because if I change the `errors` file, the validator tests rightly fails with 

```
TestRunnerTest > testRunner(String, Callable) > software.amazon.smithy.protocols.traits.TestRunnerTest.testRunner(String, Callable)[2] FAILED
    software.amazon.smithy.model.validation.testrunner.SmithyTestCase$Error: =======================
    Model Validation Result
    =======================
    /home/bigo/github/smithy/smithy-protocols-traits/build/resources/test/software/amazon/smithy/protocols/traits/errorfiles/cbor-wire-format.smithy

    Encountered unexpected events
    -----------------------------
    [ERROR] smithy.example#InvalidService1: Error validating trait `smithy.protocols#rpcv2`: Missing required structure member `format` for `smithy.protocols#rpcv2` | TraitValue N/A:0:0
        at app//software.amazon.smithy.model.validation.testrunner.SmithyTestCase$Result.unwrap(SmithyTestCase.java:277)
        at app//software.amazon.smithy.model.validation.testrunner.SmithyTestSuite.lambda$parameterizedTestSource$0(SmithyTestSuite.java:113)
        at app//software.amazon.smithy.protocols.traits.TestRunnerTest.testRunner(TestRunnerTest.java:28)
```